notebooks/en/llm_judge.ipynb

```diff
@@ -0,0 +1,716 @@
+{
```

Comment:
<p>A small typo: "This means that our "ground truth" contains noise: hence we cannot expect any algorithmic evaluation to come that close to it<strong> &nbsp;a &nbsp;</strong>However, we could reduce a bit this noise: " =&gt; </p><p>"This means that our "ground truth" contains noise: hence we cannot expect any algorithmic evaluation to come that close to it. However, we could reduce this noise <strong>a bit</strong>: "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>